CONTRIBUTING.md

```diff
@@ -353,7 +353,7 @@ So please run `make lint` instead.
 
 NOTE: This use of devcontainers for terraform-provider-databricks development is **experimental** and not officially supported by Databricks
 
-This project has configuration for working with [Visual Studio Code Devcontainers](https://code.visualstudio.com/docs/remote/containers) - this allows you to containerise your development rerequisites (e.g. golang, terraform). To use this you will need [Visual Studio Code](https://code.visualstudio.com/) and [Docker](https://www.docker.com/products/docker-desktop).
+This project has configuration for working with [Visual Studio Code Devcontainers](https://code.visualstudio.com/docs/remote/containers) - this allows you to containerize your development prerequisites (e.g. golang, terraform). To use this you will need [Visual Studio Code](https://code.visualstudio.com/) and [Docker](https://www.docker.com/products/docker-desktop).
```

Comment:
could this change be separated in a different PR?

Reply:
Yep, handled this in #3512